api/admin/controller/__init__.py

```diff
@@ -937,13 +937,16 @@ def _create_or_update_list(self, library, name, entries, collections, deletedEnt
         list.name = name
         membership_change = False
 
+        works_to_update_in_search = []
```

Comment:
It doesn't matter much, but this should be a set.

Reply:
Good call.